services/collaboration/pkg/connector/fileconnector.go

```diff
@@ -281,7 +300,7 @@ func (f *FileConnector) Lock(ctx context.Context, lockID, oldLockID string) (*Co
 			logger.Warn().
 				Str("LockID", resp.GetLock().GetLockId()).
 				Msg("SetLock lock refreshed instead")
-			return NewResponse(200), nil // no need to send the lockID for a 200 code
+			return NewResponseWithVersionAndLock(200, statResp.GetInfo().GetMtime(), resp.GetLock().GetLockId()), nil
```

Comment:
According to the previous comment, the lockID shouldn't be needed for the 200 response. It also contradicts the official MS docs (https://learn.microsoft.com/en-us/microsoft-365/cloud-storage-partner-program/rest/files/lock#response-headers). If it's needed, we should add a comment to explain the situation.

Reply:
I see, will change it.